relentless/potential/pair.py

```diff
@@ -6,10 +6,57 @@
 from .potential import PairPotential
 
 class LennardJones(PairPotential):
+    """Lennard-Jones 12-6 pair potential.
+
+    .. math::
+
+    u(r) = 4 \varepsilon\left[\left(\frac{\sigma}{r}\right)^12 - \left(\frac{\sigma}{r}\right)^12 \right]
+
+    The required coefficients per pair are:
+
+    - :math:`\varepsilon`: interaction energy
+    - :math:`\sigma`: interaction length scale (e.g., particle diameter)
+
+    The optional coefficients per pair are:
+
+    - ``rmin``: minimum radius, energy and force are 0 for ``r < rmin``. Ignored if ``False`` (default).
+    - ``rmax``: maximum radius, energy and force are 0 for ``r > rmax`` Ignored if ``False`` (default).
+    - ``shift``: If ``True``, shift potential to zero at ``rmax`` (default is ``False``).
+
```

Comment:
I forgot to add this earlier, but could you add the following comment here:
```py
Setting ``rmax = sigma*2**(1./6.)`` and ``shift = True`` will give the purely repulsive Weeks-Chandler-Andersen potential, which can model nearly hard spheres.
```
(separate from the list above by a blank line)